core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -284,21 +258,25 @@ public long getAvailableBytes() {
    * @return the capacity of the worker in bytes
    */
   public long getCapacityBytes() {
-    return mCapacityBytes;
+    return mUsage.mCapacityBytes;
```

Comment:
does this and such methods require locking outside?
can you update comments if so?

Reply:
Added comments to this method and the neighbors that require external locking. (Looking a bit verbose now :/)